src/WSRequestHandler_SceneCollections.cpp

```diff
@@ -12,26 +17,32 @@
  * @category scene collections
  * @since 4.0.0
  */
-RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest& request) {
-	if (!request.hasField("sc-name")) {
+RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest &request)
+{
+	if (!request.hasField("sc-name"))
+	{
```

Comment:
The opening bracket of `if` blocks must be on the same line as the statement.

Reply:
Sorry, I left my auto-formatter enabled and it was configured for a different coding style. I reverted all the code "changes" made by it and left only the relevant one that is in the comment at the beginning of the file.
Excuse me for the waste of time due to my distraction.